figure.html

```diff
@@ -188,17 +240,30 @@
 
 </li>
 
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
 <li>
-	<p class="first">Click on the <span class="bold">Projection</span> icon to display the image as a maximum intensity view.</p>
-<p>The range of Z sections included in the projection is displayed below the slider.</p>
-<p>Increment or decrement the Z sections to shift the range of sections included in the projection.</p>
-<p>Expand the Z section range using the drag handles on the slider.</p>
+<p class="first">Click on the crop icon to open the <span class="bold">Crop to Region</span> panel.</p>
+
+<p>Select an existing ROI to use as a cropping region,</p>
+
+<p>or click and drag on the image to draw a crop region.</p>
+
+<p>Use the drag handles to adjust the crop and click <span class="bold">OK</span>.p>
```

Comment:
Missing </ on your </p>. I have made some minor UI changes to the crop dialog, adding a short info message. Probably not a big enough change to require new screenshots.